src/commands/registry.rs

```diff
@@ -6,16 +6,13 @@ use std::{io::Read, path::PathBuf};
 use crate::opts::*;
 
 /// Commands for working with OCI registries to distribute applications.
-/// The set of commands for OCI is EXPERIMENTAL, and may change in future versions of Spin.
-/// Currently, the OCI commands are reusing the credentials from ~/.docker/config.json to
-/// authenticate to registries.
 #[derive(Subcommand, Debug)]
 pub enum RegistryCommands {
     /// Push a Spin application to a registry.
     Push(Push),
     /// Pull a Spin application from a registry.
     Pull(Pull),
-    /// Log in to a registry.
+    /// Log in to a registry using credentials from ~/.docker/config.json.
```

Comment:
Looking at the `oci::auth` module I think even this no longer uses the Docker credentials, except maybe as a fallback?

```
    fn default_path() -> Result<PathBuf> {
        Ok(dirs::config_dir()
            .context("Cannot find configuration directory")?
            .join("fermyon")
            .join("registry-auth.json"))
    }
```

I'd suggest removing the auth file path altogether.

Reply:
Yeah you beat me to the words. I agree. Thanks!